src/alitra/frame_transform.py

```diff
@@ -15,17 +24,30 @@ class FrameTransform:
 
     def __init__(
         self,
-        euler: Euler,
         translation: Translation,
         from_: Literal["robot", "asset"],
         to_: Literal["robot", "asset"],
+        euler: Euler = None,
+        quaternion: Quaternion = None,
```

Comment:
We can discuss this further, but Im thinking we make a Rotation class / interface, where both Euler and Quaternion implements the Rotation class, or where the Rotation class can be initiates as either a euler rotation or quaternion.

Reply:
Agreed. I will leave to a future PR though.